package_control/download_manager.py

```diff
@@ -1,8 +1,10 @@
+import os
+from pathlib import Path
```

Comment:
pathlib is not a possible dependency to rely on in Package Control 4.x as it is not available on python 3.3. Vendoring for just this single task is not desired.